codebuild/common-windows.bat

```diff
@@ -12,5 +12,6 @@ set AWS_C_INSTALL=%cd%\build\deps\install
 python setup.py build install
 python elasticurl.py -v ERROR -P -H "content-type: application/json" -i -d "{'test':'testval'}" http://httpbin.org/post
 python elasticurl.py -v ERROR -i https://example.com
+python -m unittest tests.server_test
```

Comment:
If this were just `python -m unittest` then it would also automatically discover and run any future unit tests that we add.